manifests/foreman_proxy.pp

```diff
@@ -107,13 +107,26 @@
       require    => Cert[$proxy_cert_name],
     }
 
-    file { $proxy_ca_cert:
-      ensure  => file,
-      source  => $default_ca_cert,
+    concat { $proxy_ca_cert:
+      ensure  => present,
       owner   => $owner,
       group   => $group,
       mode    => '0440',
-      require => File[$default_ca_cert],
+      require => File[$server_ca_cert],
```

Comment:
This require is a bit odd. Wouldn't it be on the individual fragments? I'm not sure if it does that automatically (I'm guessing not) so be sure to check that.